frontend/src/shims-vue.d.ts

```diff
@@ -4,3 +4,8 @@ declare module '*.vue' {
   const component: DefineComponent<{}, {}, any>
   export default component
 }
+
+declare module '*.md' {
```

Comment:
Can I double check what this is for?

Reply:
This used to be for importing markdown files into Vue components, but now that it's no longer being imported at compile time, it's no longer needed. I'll remove it!